client/.storybook/environmentDecorator.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import GlobalStyles from '../src/GlobalStyles';
+import styled from '@emotion/styled';
+
+const AppContainer = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
```

Comment:
Does this one need to be a `flex`? It seems like `MainContainer` will be the only child anyway

Reply:
yeah they need to be in this PR because I removed the manually applied AppContainer and MainContainer there 😉 

and yeah it has to be flex later because actually this storybook decorator is not 100% realistic since in my App I will have the side and bottom nav outside of the main container and only inside the App one... however I couldn't figure out how to apply these conditions to the storybook decorator so I left it like this.... 🤔